docker/DockerFile.dev

```
@@ -0,0 +1,26 @@
+# check docker/Readme.md
+
+FROM ubuntu:impish
```

```suggestion
FROM debian:stable
```
Impish will fall out of supported status in a couple months.